docs/guides/dataset/configure_child_chunking_strategy.md

```
@@ -33,4 +33,4 @@ Through this dual-layer structure of "precise localization + contextual suppleme
 
 3. This configuration applies to the **Chunker** component when it comes to ingestion pipeline settings:
 
-![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/child_chunking_chunker.png)
\ No newline at end of file
+![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/child_chunking_chunker.png)
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)